shotover-proxy/src/transforms/cassandra/connection.rs

```diff
@@ -40,7 +40,9 @@ impl CassandraConnection {
         mut tls: Option<TlsConnector>,
         pushed_messages_tx: Option<mpsc::UnboundedSender<Messages>>,
     ) -> Result<Self> {
-        let tcp_stream: TcpStream = TcpStream::connect(&host).await?;
+        let tcp_stream = timeout(Duration::from_secs(3), TcpStream::connect(&host))
+            .await?
+            .map_err(|e| anyhow::Error::new(e).context("Failed to connect to upstream"))?;
```

Comment:
I think we should include the `host` value in the error message, that should make it a lot easier to track down connection issues to specific nodes.